app/3.0/start/toolbox/deploy.md

```diff
@@ -88,31 +138,31 @@ and create a new project.
     threadsafe: yes
 
     handlers:
-    - url: /bower_components
-      static_dir: build/es5-bundled/bower_components
+    - url: /node_modules
+      static_dir: build/es6-bundled/node_modules
       secure: always
 
     - url: /images
-      static_dir: build/es5-bundled/images
+      static_dir: build/es6-bundled/images
       secure: always
 
     - url: /src
-      static_dir: build/es5-bundled/src
+      static_dir: build/es6-bundled/src
       secure: always
 
     - url: /manifest.json
-      static_files: build/es5-bundled/manifest.json
-      upload: build/es5-bundled/manifest.json
+      static_files: build/es6-bundled/manifest.json
+      upload: build/es6-bundled/manifest.json
       secure: always
 
     - url: /service-worker.js
-      static_files: build/es5-bundled/service-worker.js
-      upload: build/es5-bundled/service-worker.js
+      static_files: build/es6-bundled/service-worker.js
+      upload: build/es6-bundled/service-worker.js
       secure: always
 
     - url: /.*
-      static_files: build/es5-bundled/index.html
-      upload: build/es5-bundled/index.html
+      static_files: build/es6-bundled/index.html
+      upload: build/es6-bundled/index.html
       secure: always
     ```
```

Comment:
Suggest you add `skip_files` to app.yaml to avoid uploading unused files (we are doing this to our pwa-starter-kit):

```
skip_files:
- build/es5-bundled/
- build/esm-bundled/
- images/
- node_modules/
- src/
- test/
```

(if necessary, replace `build/es5-bundled/` with whatever build _isn't_ used above)

Reply:
Yes, this is definitely critical! Just did this and found that it uploaded 26000 objects thanks to `node_modules/`, which is greater than the 10000 max allowed to deploy!

Reply:
I wonder if we should tell users to put this file in the `build/es5-bundled/` folder itself and remove the folder prefixes here, then they wouldn't need to have `skip_files` and it wouldn't upload any extra files at all.

Reply:
Actually, I suppose that would upload `app.yaml` itself, which would probably be bad.

Reply:
`polymer build` cleans out the build/ directory. You could add app.yaml to `extraDependencies` in polymer.json and remember to deploy from build/es5-bundled or run `gcloud deploy build/es5-bundled/app.yaml`

Reply:
You have to upload app.yaml anyways, and it wouldn't be served since there's no handlers that point to it.

Reply:
I am ok with leaving app.yaml where it is as long as it works..